Extension/walkthrough/run-and-debug-project-mac.md

```diff
@@ -0,0 +1,13 @@
+# Run and debug your C++ file on macOS
+
+1. Open the C++ source file that you want to run and debug. Make sure this file is active (currently displayed and selected) in the editor.
+
+2. From the main menu, select **Run > Add Configuration...** and choose **C++ (GDB/LLDB)**.
+
+3. Choose **clang++ build and debug active file.**
+
+    ![Dropdown showing C++ debug configurations for Clang on macOS](clang-build-and-debug-active-file.png)
+
+4. To start a debugging session, Press `F5` or from the main menu choose **Run > Start Debugging**.
```

Comment:
`Press` for "# Run and debug your C++ file on macOS" and "# Run and debug your C++ file on Windows" also needs to be lowercased.

Reply:
Oh true, thanks!